src/prefect/utilities/graphql.py

```diff
@@ -275,3 +277,33 @@ def with_args(field: Any, arguments: Any) -> str:
     parsed_field = parse_graphql(field)
     parsed_arguments = parse_graphql_arguments(arguments)
     return "{field}({arguments})".format(field=parsed_field, arguments=parsed_arguments)
+
+
+def compress_dict_to_str(dictionary: dict) -> str:
```

Comment:
Sorry to keep holding this up, but I think the name / types here are overly specific --> the code as written can handle any json serializable object (numbers, strings, lists, etc.)